README.md

```diff
@@ -7,6 +7,7 @@ This repository contains code and documentation for use with
  * `codelabs/opencv-haarcascade` provides the source code for the [OpenCV Dataproc Codelab](https://codelabs.developers.google.com/codelabs/cloud-dataproc-opencv/index.html), which demonstrates a Spark job that adds facial detection to a set of images. 
 * `codelabs/spark-bigquery` provides the source code for the [PySpark for Preprocessing BigQuery Data  Codelab](https://codelabs.developers.google.com/codelabs/pyspark-bigquery/index.html), which demonstrates using PySpark on Cloud Dataproc to process data from BigQuery.
 * `codelabs/spark-nlp` provides the source code for the [PySpark for Natural Language Processing Codelab](https://codelabs.developers.google.com/codelabs/spark-nlp/index.html), which demonstrates using [spark-nlp](https://github.com/JohnSnowLabs/spark-nlp) library for Natural Language Processing.
+* `notebooks/ai-ml/ImageClassificationInSpark.py` provides source code for using [PyTorch](https://pytorch.org/) with Apache Spark for image classification.
```

Comment:
`notebooks/ai-ml` We should just give description of this folder where other notebooks will be added